docs/modules/develop/pages/classes/forms.adoc

```diff
@@ -0,0 +1,28 @@
+= Forms
+
+This is a form example that can be implemented in your application.
+
+Form classes are located in the `app/forms/decidim/<my_plugin>` directory, and named: `<my_event>_form.rb`.
+
+```ruby
+module Decidim
```

Comment:
Missing frozen string and filename here. All the examples in this page should have them

Reply:
Fixed in c010e4fed7